src/mame/nmk/nmk214.h

```diff
@@ -0,0 +1,38 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergio Galiano
+#ifndef MAME_NMK_NMK214_H
+#define MAME_NMK_NMK214_H
+
+#pragma once
+
+
+class nmk214_device : public device_t
+{
+public:
+	nmk214_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	void set_mode(u8 mode) { m_mode = mode; }
+	void set_init_config(u8 init_config);
+	bool is_device_initialized() { return m_device_initialized; }
+	void set_input_address_bitswap(const u8 *input_address_bitswap) { m_input_address_bitswap = input_address_bitswap; }
```

Comment:
Please visually separate the setters for hard-coded things (the hard-wired mode and the address scrambling) from things that can change dynamically.

Reply:
Done